src/llama.cpp

```diff
@@ -12825,6 +12909,216 @@ struct llm_build_context {
         return gf;
     }
 
+    struct ggml_cgraph * build_minicpm3() {
+        struct ggml_cgraph * gf = ggml_new_graph_custom(ctx0, llama_model_max_nodes(model), false);
+
+        int32_t n_tokens = this->n_tokens;
```

Comment:
```suggestion
```

This is not necessary (`n_tokens` is not mutated). Removing this makes `n_tokens` instead directly refer to `this->n_tokens`.

Reply:
Fixed.